# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/11/27 23:02:48+00:00 dwmw2@shinybook.infradead.org 
#   MTD: Fix timing setup for NAND flash on Samsung S3C2410.
#   
#   Spotted by Shannon Holland.
#   
#   Signed-off-by: Ben Dooks <ben@simtec.co.uk>
#   Signed-off-by: David Woodhouse <dwmw2@infradead.org>
# 
# drivers/mtd/nand/s3c2410.c
#   2004/11/27 23:02:28+00:00 dwmw2@shinybook.infradead.org +2 -2
#   revision 1.6
#   date: 2004/11/24 12:25:48;  author: bjd;  state: Exp;  lines: +2 -2
#   correct timing setup to use plat->twrph1 instead of
#   plat->twrph0 for timing setup for the NAND controllers
#   twrph1 configuration
#   
#   Thanks to Shannon Holland for pointing this out
# 
diff -Nru a/drivers/mtd/nand/s3c2410.c b/drivers/mtd/nand/s3c2410.c
--- a/drivers/mtd/nand/s3c2410.c	2005-02-14 05:15:46 -08:00
+++ b/drivers/mtd/nand/s3c2410.c	2005-02-14 05:15:46 -08:00
@@ -11,7 +11,7 @@
  *	28-Sep-2004  BJD  Fixed ECC placement for Hardware mode
  *	12-Oct-2004  BJD  Fixed errors in use of platform data
  *
- * $Id: s3c2410.c,v 1.5 2004/10/12 10:10:15 bjd Exp $
+ * $Id: s3c2410.c,v 1.6 2004/11/24 12:25:48 bjd Exp $
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -167,7 +167,7 @@
 	if (plat != NULL) {
 		tacls = s3c2410_nand_calc_rate(plat->tacls, clkrate, 8);
 		twrph0 = s3c2410_nand_calc_rate(plat->twrph0, clkrate, 8);
-		twrph1 = s3c2410_nand_calc_rate(plat->twrph0, clkrate, 8);
+		twrph1 = s3c2410_nand_calc_rate(plat->twrph1, clkrate, 8);
 	} else {
 		/* default timings */
 		tacls = 8;
